app/javascript/CommentSubscription/CommentSubscription.jsx

```diff
@@ -136,7 +136,7 @@ export class CommentSubscription extends Component {
           <Dropdown
             className={
               showOptions
-                ? `inline-block z-10 right-4 left-4 s:right-0 s:left-auto${
+                ? `inline-block z-30 right-4 left-4 s:right-0 s:left-auto${
```

Comment:
I had to bump this to `z-30` @ludwiczakpawel because on mobile the sticky reactions bar was on top of it.

![image](https://user-images.githubusercontent.com/833231/80777685-05cb6f00-8b34-11ea-9fbe-a45b9786a832.png)

Reply:
👍 As you mentioned in some other PR - all our z-indexes need to be audited at some point :)